deployments/charts/kuma/README.md

```diff
@@ -7,57 +7,60 @@ The chart supports Helm v3+.
+| `controlPlane.kdsGlobalAddress`                    | URL of Global Kuma CP                                                                               |                                                          |
+| `controlPlane.nodeSelector`                        | Node Selector for the Kuma Control Plane pods                                                       | `{ kubernetes.io/os: linux, kubernetes.io/arch: amd64 }` |
+| `controlPlane.injectorFailurePolicy`               | Failure policy of the mutating webhook implemented by the Kuma Injector component                   | `Ignore`                                                 |
+| `controlPlane.service.name`                        | Service name of the Kuma Control Plane                                                              | nil                                                      |
+| `controlPlane.service.type`                        | Service type of the Kuma Control Plane                                                              | ClusterIP                                                |
+| `controlPlane.service.annotations`                 | Additional annotations to put on the Kuma Control Plane service                                     | {}                                                       |
+| `controlPlane.globalRemoteSyncService.name`        | Service name of the Global-Remote Sync                                                              | nil                                                      |
+| `controlPlane.globalRemoteSyncService.type`        | Service type of the Global-Remote Sync                                                              | LoadBalancer                                             |
+| `controlPlane.globalRemoteSyncService.port`        | Port on which Global-Remote Sync is exposed                                                         | 5685                                                     |
+| `controlPlane.globalRemoteSyncService.annotations` | Additional annotations to put on the Global-Remote Sync service                                     | {}                                                       |
+| `controlPlane.defaults.skipMeshCreation`           | Whether or not to skip creating the default Mesh                                                    | `true`                                                   |
+| `controlPlane.resources`                           | The K8s resources spec for Kuma CP                                                                  | nil, differs based on mode                               |
+| `controlPlane.tls.general.secret`                  | Secret that contains tls.crt, key.crt and ca.crt for protecting Kuma in-cluster communication       | nil, generated and self-signed                           |
```

Comment:
nit, what do you think about `.secretName` or `.existingSecretName`? I've seen that pattern used more often, ex in [bitnami/rabbitmq](https://artifacthub.io/packages/helm/bitnami/rabbitmq#exposure-parameters)

Reply:
I was thinking whether to pick `.secret` or `.secretName`, but I settled down on `.secret`.

You definitely have more experience with HELM charts, so I'll trust your intuition here - changed to `.secretName`.
Thanks for the review!